contrib/gitian/DOCKRUN.md

```diff
@@ -57,7 +57,7 @@ The dockrun.sh script will do everything to build the binaries. Just specify the
 version to build as its only argument, e.g.
 
 ```bash
-./dockrun.sh v0.17.3.0
+./dockrun.sh v0.18.1.0
```

Comment:
Changes to this file should be made after https://github.com/monero-project/monero/pull/8442 is merged.

Reply:
The PR you linked is against master branch and this one is against release branch.

Reply:
Should we get seth to submit that against release branch then? Pointless to do these edits redundantly.